src/main/java/com/google/sps/data/TagRecipePair.java

```diff
@@ -19,16 +19,19 @@
  * Class representing recipe tag
  */
 public class TagRecipePair {
+  private long tagId;
```

Comment:
Can tag be defined as a enum type?

Reply:
I don't think tag can be defined as a enum type because the tag name can be whatever the user sets the tag name to be, therefore trying to use predefined constants for tags would be difficult unless we decide to have the user only use certain tags that we define beforehand.